libs/feature/record/src/lib/record-metadata/record-metadata.component.ts

```diff
@@ -55,6 +56,11 @@ export class RecordMetadataComponent {
   errorTypes = ErrorType
   selectedTabIndex$ = new BehaviorSubject(0)
 
+  lightbox$ = this.facade.metadata$.pipe(
+    map((metadata) => metadata?.thumbnailUrl)
+  )
+  lightboxInstance: any
```

Comment:
Should this stay `any`?

Reply:
Fixed ! Removed it and set it as const in method.